e2e/helper/apps.list.js

```diff
@@ -0,0 +1,75 @@
+export const deployment = [
+    "1. Create a Namespace",
+    "2. Connect & Configure",
+    "3. Deploy & Monitor"
+];
+
+export const deployDesc = [
+    "Reserve a unique path for your app and add it to One Platform application catalog.",
+    "Connect your app to One Platform Micro-service APIs from Developer Console.",
+    "Easily deploy your app using SPAship and monitor app performance, uptime and analytics."
+];
+
+export const microservicesHeader = [
+    "Authentication",
+    "Feedback",
+    "Notification",
+    "User Group",
+    "Search"
+];
+
+export const microservicesDesc = [
+    "Adding authentication to your application is pice of cake. Just integrate with our API and focus on application development.",
+    "Allows users to review and rate services and products provided by teams. Built into a power packed microservice so serve all.",
+    "Notification framework for One Platform and also all other SPAs",
+    "A prebuilt integration with LDAP to authenticate, authorize and get user information.",
+    "A unified search API gives you ability to search across all hosted applications on the platform."
+];
+
```

Comment:
WHy this content is needed. this is specific to one platform. i suggest to do a clean up